c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -1259,6 +1260,26 @@ namespace CacheOverride {
   }
 }
 
+
+// https://fetch.spec.whatwg.org/#concept-method-normalize
+// Returns `true` if the method name was normalized, `false` otherwise.
+static bool normalize_http_method(char* method) {
+  static const char* names[6] = { "DELETE", "GET", "HEAD", "OPTIONS", "POST", "PUT" };
+
+  for (size_t i = 0; i < 6; i++) {
+    auto name = names[i];
+    if (strcasecmp(method, name) == 0) {
+      if (strcmp(method, name) == 0) {
+        return false;
+      }
+      strcpy(method, name);
```

Comment:
For the benefit of any auditors in the future, can we add a safety comment here noting that `name` and `method` have the same length (because they compare equal modulo case) so `strcpy` is safe?